helm/resource_release.go

```diff
@@ -1186,6 +1218,34 @@ func getValues(d resourceGetter) (map[string]interface{}, error) {
 	return base, logValues(base, d)
 }
 
+func getListValue(base, set map[string]interface{}) error {
+	name := set["name"].(string)
+	listValue := set["value"].([]interface{}) // this is going to be a list
+	valueType := set["type"].(string)
+
+	listStringArray := make([]string, len(listValue))
+
+	for i, s := range listValue {
+		listStringArray[i] = s.(string)
+	}
+	listString := strings.Join(listStringArray, ",")
+
+	switch valueType {
+	case "auto", "":
+		if err := strvals.ParseInto(fmt.Sprintf("%s={%s}", name, listString), base); err != nil {
+			return fmt.Errorf("failed parsing key %q with value %s, %s", name, listString, err)
+		}
+	case "string":
+		if err := strvals.ParseIntoString(fmt.Sprintf("%s={%s}", name, listString), base); err != nil {
+			return fmt.Errorf("failed parsing key %q with value %s, %s", name, listString, err)
+		}
+	default:
+		return fmt.Errorf("unexpected type: %s", valueType)
+	}
+
+	return nil
```

Comment:
Because this is a list and not individual values I think we can actually just use `ParseInto()` here. It only makes a difference when you are parsing individual values that might be numbers or bools.

Reply:
You'll need to remove `type` from the schema too.